### ISSUE TITLE
Safe fix for `FURB180` can delete comments

### ISSUE THREAD

**Issue by ntBre** [astral-sh/ruff#22631]
This comment will be deleted:

```py
import abc


class C(
    other_kwarg=1,
    # comment
    metaclass=abc.ABCMeta,
):
    pass
```

https://play.ruff.rs/f64dfdc1-811d-466f-8e49-0f400e1859d0

<hr>

We should make the fix unsafe here if there are comments in the deletion range, as well as when there are other base classes:

https://github.com/astral-sh/ruff/blob/b80d8ff6fffa3b9f3c54877f21775c2ba958ba63/crates/ruff_linter/src/rules/refurb/rules/metaclass_abcmeta.rs#L85-L89

and update the fix safety docs to reflect the other way the fix can be unsafe:

https://github.com/astral-sh/ruff/blob/b80d8ff6fffa3b9f3c54877f21775c2ba958ba63/crates/ruff_linter/src/rules/refurb/rules/metaclass_abcmeta.rs#L41-L44

**Comment by AndyBodnar** [comment]
I'll take this one.

**Comment by ntBre** [comment]
Sorry @AndyBodnar, this is already resolved by the linked PR #22234.
